.github/PULL_REQUEST_TEMPLATE/manifest_add.md

```diff
@@ -0,0 +1,10 @@
+---
+name: New manifest
+about: Add new manifest into bucket.
+title: 'Add %%manifestname%% %%manifestversion%%'
+labels: please-review
+---
+
```

Comment:
```suggestion

<!-- Please run .\bin\checkver.ps1 at least once to make sure the auto-update feature works -->
- [ ] Auto Update works
- [ ] ...
```

Should we add a checklist? With common tasks, like adding a description and stuff?

Reply:
I would go with:

1. Description
1. Autoupdate
1. Installation
    1. 64bit
    1. 32bit

While i am doing static manifest review i require only description property (next to installation required properties of course)
So for me are installation and autoupdate enough. (autoupdate will preserve that hash will be corrected, manifest is formatted and it is valid json)

Reply:
And depending on users input is not good move. As soon as Actions are deployed it will be automatic.

https://github.com/Ash258/GithubActionsBucketForTesting/pull/111
https://github.com/Ash258/GithubActionsBucketForTesting/runs/178856674